CHANGELOG.md

```diff
@@ -41,6 +43,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Deprecated
 
 ### Removed
+- **UIView**
+  - Removed enums `ShakeDirection`, `AngleUnit`, `ShakeAnimationType` and their dependent functions.  [#756](https://github.com/SwifterSwift/SwifterSwift/pull/756) by [mmdock](https://github.com/mmdock)
```

Comment:
This means we have a breaking change, technically. Shouldn't be including this until the next major version.

Reply:
@guykogus yeah, i was thinking maybe i just  add those back and mark deprecated

Reply:
I think that would be better. We actually have a `Deprecated` folder for each section, although currently there's only `StdlibDeprecated.swift`. You'd need to move these to `UIViewDeprecated.swift`.